### ISSUE TITLE
misp-modules server fails to start with Python 3.6 due to missing module dnspython3

### ISSUE THREAD

**Issue by chrisinmtown** [MISP/misp-modules#463]
tl;dr - **Are these packages just missing from the REQUIREMENTS file or is there a deeper problem?** 

I'm using Python 3.6 with the misp-modules version tagged v2.4.134 on my first attempt at installing and running. I just keep finding missing dependencies.  I gave up on expansion/farsight_passivedns.py because of the dndb2/importlib snafu, simply deleted the module and the entry in misp_modules/modules/expansion/__init__.py. I start the system like this:
```
$ /var/www/MISP/venv/bin/misp-modules -l 127.0.0.1 -s
```
So far I have hit the following missing packages.  I install each package in the venv, then retry.  So far the list includes:
```
censys
socialscan
```
Then I hit this one which is confusing:
```
-bash-4.2$ /var/www/MISP/venv/bin/misp-modules -l 127.0.0.1 -s
Unable to import URLObject, pyfaup missing
```
I found that string here:

https://github.com/MISP/misp-modules/blob/7104a35cca7faee027fcb198b3a1bd49796ef0a3/misp_modules/modules/import_mod/email_import.py#L13

So I installed package pyfaup and WHEW the server came up!

**Comment by chrisinmtown** [comment]
I tested install&launch by creating new virtual environments with different Python versions. Here's what I can offer:

* Python 3.6, pip 20.3.3: the misp-modules server failed to start due to missing module dnspython3
* Python 3.7, pip 20.3.3: server starts without complaint 
* Python 3.8, pip 20.3.3: server starts without complaint

I believe the REQUIREMENTS file is deficient for 3.6, I propose this change to that file.  This change seems to work in the 3.7 and 3.8 scenarios also:
```
diff --git a/REQUIREMENTS b/REQUIREMENTS
index 10fb179..019efef 100644
--- a/REQUIREMENTS
+++ b/REQUIREMENTS
@@ -37,7 +37,7 @@ cryptography==3.2.1
 decorator==4.4.2
 deprecated==1.2.10; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
 dnsdb2==1.1.1
-dnspython==2.0.0
+dnspython3==1.15.0
 domaintools-api==0.5.2
 enum-compat==0.0.3
 ez-setup==0.9
```

**Comment by chrisinmtown** [comment]
Is python3.6 just too old?  It's depressing that 4 months have passed since I opened this, and nobody from CIRCL has commented at all.

**Comment by chrisinmtown** [comment]
Thanks for the recent updates to the REQUIREMENTS file.  I cloned the misp-modules repo and installed from the tip of branch main, commit 2bc5021ac (I see no tags). I installed python packages successfully using Python 3.6. The server came up successfully.
